### ISSUE TITLE
Cannot read property 'nodeValue' of undefined

### ISSUE THREAD

**Issue by petermlyon** [opencv/cvat#1117]
I'm experiencing an issue that is happening a lot. I am attempting to annotate polygons. I can't find what exactly triggers the issue to happen, but it seems to happen more regularly the further into a data set the annotator is.

```
fe4c4436ce4b.js:8743 Uncaught TypeError: Cannot read property 'nodeValue' of undefined
    at fe4c4436ce4b.js:8743
    at Array.sort (<anonymous>)
    at Function.sortByZOrder (fe4c4436ce4b.js:8742)
    at ShapeCollectionView.onCollectionUpdate (fe4c4436ce4b.js:8644)
    at ShapeCollectionModel.notify (fe4c4436ce4b.js:1224)
    at ShapeCollectionModel._interpolate (fe4c4436ce4b.js:7278)
    at ShapeCollectionModel.update (fe4c4436ce4b.js:7608)
    at ShapeCreatorModel.get currentShapes [as currentShapes] (fe4c4436ce4b.js:10141)
    at ShapeCreatorController.get currentShapes [as currentShapes] (fe4c4436ce4b.js:10218)
    at HTMLInputElement.<anonymous> (fe4c4436ce4b.js:10343)
fe4c4436ce4b.js:11871 Uncaught Error: TypeError: Cannot read property 'nodeValue' of undefined
    at AnnotationSaverModel.save (fe4c4436ce4b.js:11818)
```

```
Uncaught TypeError: Cannot read property 'nodeValue' of undefined
    at fe4c4436ce4b.js:8743
    at Array.sort (<anonymous>)
    at Function.sortByZOrder (fe4c4436ce4b.js:8742)
    at PolygonView.onShapeUpdate (fe4c4436ce4b.js:6398)
    at PolygonView.onShapeUpdate (fe4c4436ce4b.js:6850)
    at PolygonModel.notify (fe4c4436ce4b.js:1224)
    at PolygonModel.notify (fe4c4436ce4b.js:3991)
    at PolygonModel.set active [as active] (fe4c4436ce4b.js:4301)
    at ShapeCollectionModel.resetActive (fe4c4436ce4b.js:7613)
    at ShapeCollectionModel._clear (fe4c4436ce4b.js:7258)
```

It appears that the sortByZIndex() function is somehow being called on uninitialized items. This happens both when making new shapes and modifying existing ones.

**Comment by bsekachev** [comment]
@petermlyon 

We met this issue and it was fixed #1068.
Try to update CVAT.

**Comment by petermlyon** [comment]
Thanks @bsekachev, will open a new issue if it persists.
